pkgs/development/python-modules/stm32loader/default.nix

```
@@ -0,0 +1,33 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, progress
+, pyserial
+, pytest
+}:
+
+buildPythonPackage rec {
+  pname = "stm32loader";
+  version = "0.5.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1w6jg4dcyz6si6dcyx727sxi75wnl0j89xkiwqmsw286s1y8ijjw";
+  };
+
+  propagatedBuildInputs = [ progress pyserial ];
+
+  checkInputs = [ pytest ];
```

I noticed test would fail on python 2.7 with
` ImportError: No module named mock`

```suggestion
  checkInputs = [ pytest ] ++ lib.optional isPy27 mock;
```

> https://github.com/florisla/stm32loader/blob/master/tests/unit/test_bootloader.py#L12